package.json

```diff
@@ -21,24 +22,22 @@
   ],
   "license": "MIT",
   "files": [
-    "dist",
-    "index.d.ts"
+    "dist"
   ],
+  "prettier": "@github/prettier-config",
   "devDependencies": {
-    "@babel/core": "^7.6.2",
-    "babel-preset-github": "^3.2.1",
+    "@github/prettier-config": "0.0.4",
     "chai": "^4.2.0",
-    "eslint": "^6.5.1",
-    "eslint-plugin-github": "^3.1.3",
-    "flow-bin": "^0.111.1",
-    "karma": "^4.3.0",
+    "eslint": "^6.8.0",
+    "eslint-plugin-github": "^3.4.1",
```

Comment:
I think updating to 4.0.0 here would be good.

Reply:
Does https://github.com/github/clipboard-copy-element/pull/30/commits/f5161d84f6a6453f425d250d6fe746ba01a3508d look right to you?

Reply:
Looks like it is now also causing lint failures in Windows 🤯. I'm gonna break this change into a separate branch instead.